dev_guide/deployments.adoc

```diff
@@ -10,27 +10,35 @@
 toc::[]
 
 == Overview
-A deployment in OpenShift is an update to a single replication controller's pod
-template based on triggered events. The `*deployment*` subsystem provides:
 
-- A link:#defining-a-deploymentConfig[deploymentconfig] that drives automated deployments by the system.
-- link:#triggers[Triggers] which drive new deployments in response to events.
-- User-customizable link:#strategies[strategies] for deployment rollout behavior, which are responsible for making a deployment live in the cluster.
+A deployment in OpenShift is a replication controller based on a user defined
+template called a deployment configuration. Deployments are created manually
+or in response to triggered events.
+
+The deployment system provides:
+
+- A link:#creating-a-deployment-configuration[deployment configuration], which is a template for deployments.
+- link:#triggers[Triggers] that drive automated deployments in response to events.
+- User-customizable link:#strategies[strategies] for deployment rollout behavior which are responsible for making a deployment active in the cluster.
 - link:#rollbacks[Rollbacks] to a previous deployment.
 - Replication link:#scaling[scaling].
-- An audit history of deployed pod template configurations.
 
-The deployment configuration has a version parameter value that increases in increments of one each time a new deployment is created from that configuration. In addition, the cause of the last deployment is added to the configuration.
+The deployment configuration contains a version number that is incremented
+each time a new deployment is created from that configuration. In addition,
+the cause of the last deployment is added to the configuration.
+
+== Creating a Deployment Configuration
 
-== Defining a deploymentConfig
+A deployment configuration consists of the following key parts:
 
-A `*deploymentConfig*` describes a single link:templates.html[template] and a
-set of link:#triggers[triggers] for when a new deployment should be created.
-A deployment is simply a specially annotated `*replicationController*`.
+- A replication controller template which describes the application to be deployed.
+- The default replica count for the deployment.
+- A deployment link:#strategies[strategy] which will be used to execute the deployment.
```

Comment:
Not "execute"... but perhaps "rollout" the deployment. Or some similar word.

@adellape This would be best handled by you after this PR merges.